include/master_element/Edge32DCVFEM.h

```diff
@@ -38,6 +38,7 @@ class Edge32DSCS : public MasterElement
   using AlgTraits = AlgTraitsEdge3_2D;
 
   Edge32DSCS();
+  KOKKOS_FUNCTION
   virtual ~Edge32DSCS() {}
```

Comment:
While we are doing this cleanup can we convert the empty bodies to

```
virtual ~Edge32DSCS() = default;
``` 

and move destructor bodies in the C file to headers?